src/test/suite/didact.test.ts

```diff
@@ -264,6 +268,27 @@ suite('Didact test suite', () => {
 			}			
 		});
 	});
+
+	test('make sure we can open a didact file from a vscode extension', async () => {
+		await extensionFunctions.handleVSCodeUri(testMD5);
+		await delay(1000);
```

Comment:
better to use a waitUntil to reduce risk of flakiness and have faster test execution